docs/software-platforms/aws/starting-development-on-lsfr.md

```diff
@@ -0,0 +1,78 @@
+# Getting started with development on Lightsail for Research
+
+Once you have [created a virtual computer](creating-vcs.md), [attached a disk](working-with-volumes.md), and [accessed the operating system](creating-vcs.md#how-to-access-a-virtual-computer) you are ready to start developing analyses on Lightsail for Research.
+
+## Quick tips about virtual computers
+
+- Copy and paste need to be synced between your local and remote desktop using the clipboard icon in the upper left hand corner.
+
+    <figure markdown="span">
+        ![Clipboard](../../img/lsfr-clipboard.png){width="400"}
+    </figure>
+
+    You will also need to right-click to paste on the virtual desktop; `Ctrl+v` or `Cmd+v` will not work.
+
+- If you use a security key as a multifactor authentication method for GitHub, that will not work on a virtual computer.
+Make sure you have another way to login with multifactor authentication, such as an authenticator app.
+
+## Clone the repository
+
+Virtual computers will have Git and GitKraken installed on them.
+Click `Activities` on the upper left hand corner of the Ubuntu desktop then click the 9-dot menu to access the applications on the virtual computer.
+
+<figure markdown="span">
+    ![Activities](../../img/lsfr-activities.png){width="800"}
+</figure>
+
+<figure markdown="span">
+    ![Candy box menu](../../img/lsfr-candy-box-menu.png){width="800"}
+</figure>
+
+Open GitKraken and login using GitHub.
+
+Clone the repository to the volume you created and attached following [the setup instructions for cloning the repository](../../technical-setup/clone-the-repo.md#clone-your-forked-repository).
```

Comment:
Just a note that when setting the upstream repo I had to use the URL rather than the dropdown menu. This is different than the instructions we present in https://openscpca--349.org.readthedocs.build/en/349/technical-setup/clone-the-repo/#clone-your-forked-repository. We may want to address that here?

Reply:
I have not had this problem in any of my tests. I think we can revisit if it comes up a lot.